index.html

```diff
@@ -0,0 +1,142 @@
+     </ul>
+   </section>
+
+   <section class="main-section" id="JavaScript_and_Java">
+     <h1>JavaScript and Java</h1>
+     <p>
+       JavaScript and Java are similar in some ways but fundamentally
+       different in some others. The JavaScript language resembles Java
+       but does not have Java's static typing and strong type checking.
+       JavaScript follows most Java expression syntax, naming conventions
+       and basic control-flow constructs which was the reason why it was
+       renamed from LiveScript to JavaScript.
+     </p>
+     <p>
+       In contrast to Java's compile-time system of classes built by
+        declarations, JavaScript supports a runtime system based on a
+         small number of data types representing numeric, Boolean, and
+         string values. JavaScript has a prototype-based object model
+         instead of the more common class-based object model. The
+         prototype-based model provides dynamic inheritance; that is,
+          what is inherited can vary for individual objects. JavaScript
+           also supports functions without any special declarative requirements.
+           Functions can be properties of objects, executing as loosely
+            typed methods.
+     </p>
+     <p>
+       JavaScript is a very free-form language compared to Java.
+       You do not have to declare all variables, classes, and methods.
+        You do not have to be concerned with whether methods are public,
+         private, or protected, and you do not have to implement interfaces.
+          Variables, parameters, and function return types are not explicitly
+           typed.
+     </p>
+   </section>
+
+   <section class="main-section" id="Hello_world">
+     <h1>Hello world</h1>
+     <p>
+       To get started with writing JavaScript,
+       open the Scratchpad and write your first "Hello world" JavaScript code:
+     </p>
+     <code>function greetMe(yourName) { alert("Hello " + yourName); }</code>
+     <p>
+       Select the code in the pad and hit Ctrl+R to watch it unfold in your browser!
+     </p>
+   </section>
+
+   <section class="main-section" id="Variables">
+     <h1>Variables</h1>
+     <p>
+       You use variables as symbolic names for values in your application.
+       The names of variables, called identifiers, conform to certain rules.
+     </p>
+     <p>
+       A JavaScript identifier must start with a letter, underscore (_),
+        or dollar sign ($); subsequent characters can also be digits (0-9).
+        Because JavaScript is case sensitive, letters include the characters
+         "A" through "Z" (uppercase) and the characters "a" through "z" (lowercase).
+     </p>
+     <p>
+       You can use ISO 8859-1 or Unicode letters such as å and ü in identifiers.
+       You can also use the Unicode escape sequences as characters in identifiers.
+       Some examples of legal names are Number_hits, temp99, and _name.
+     </p>
+   </section>
+ </main>
+    
```

Comment:
The file could use a footer section here, in line with best practices👍

All in all great job

Reply:
* nicely done 👍